### ISSUE TITLE
Don't load plugins inside the butler application

### ISSUE THREAD

**Issue by keathley** [ButlerBot/butler#30]
Loading plugins this way means that when new plugins are added to someones bot they have to recompile the dependencies to see the change when running locally.

**Comment by seabre** [comment]
I ran into this. If you already compiled the app before adding a plugin.

After I hooked my plugin into `config.exs`, this is what I did to get my plugin to run: 

``` bash
mix deps.clean --all && mix deps.get && mix deps.compile && mix compile
```

After that, it recognized my plugin and I didn't have to do that again.

**Comment by keathley** [comment]
It's because we're store a configuration value inside of an attribute. The attributes value is determined at compile time and changing the configuration value doesn't cause the dependency to be re-compiled. We just need to explicitly pass in our plugins to the bot application.

**Comment by keathley** [comment]
This is fixed in 0.5.2 / 7c6b868
